pkg/apis/kubermatic/v1/admission_plugin.go

```diff
@@ -44,6 +44,7 @@ type AdmissionPlugin struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// Spec describes an admission plugin name and from which k8s version is supported.
```

Comment:
`and from which k8s version is supported.` => `and in which k8s version it is supported.`